src/App.js

```diff
@@ -18,7 +17,7 @@ const App = () => {
         <div className="container-wrapper">
           <Switch>
             <Route exact path={HOME} component={Header} />
-            <Route path={GALLERY} component={ImageGallery} />
+            <Route path={GALLERY} component={ImagesGalleryScreen} />
```

Comment:
why don't you just name it: Gallery?  it is simpler like that:
```
 <Route path={GALLERY} component={Gallery} />
```

Reply:
In the future in the future i would have two components as Images gallery and Video Gallery an it's make the sens